precommit/core.d/patchfiles.sh

```diff
@@ -327,9 +327,10 @@ function patchcmd_apply
 function patchfile_apply_driver
 {
   declare patchfile=$1
+  declare gpg=$2
 
   if declare -f ${PATCH_METHOD}_apply >/dev/null; then
-    "${PATCH_METHOD}_apply" "${patchfile}"
+    "${PATCH_METHOD}_apply" "${patchfile}" "${gpg}"
```

Comment:
I think leaving it as gpg is fine, just document that it's there and issue a warning when true for either `gitapply_apply` or `patchcmd_apply` since we can't use the signature flag in those cases (I think?, because we don't actually commit?)